packages/microcosm-preact/src/presenter.js

```diff
@@ -139,13 +138,11 @@ class PresenterMediator extends Component {
 
     this.presenter = props.presenter
     this.repo = this.presenter._requestRepo(context.repo)
-    this._lastRevision = -Infinity
```

Comment:
No need to track revisions now. When the Presenter gets new properties/state and renders, it will merge any pending model changes and cancel any delayed jobs to update the model.